docs/DocGenerator/lib/IndexData.py

```diff
@@ -0,0 +1,81 @@
+"""
+brief: Wazuh DocGenerator config parser.
```

Comment:
Modify this brief with the description of the module.

Reply:
Done